deployment/deploy_ingestion.sh

```diff
@@ -0,0 +1,112 @@
+echo "Loading available resource variables from resource.json..."
+SUFFIX=$(cat resources.json | jq .resource_suffix | tr -d '"')
+S3_READ_WRITE_POLICY=$(cat resources.json | jq .s3_read_write_policy | tr -d '"')
+CLOUDWATCH_POLICY=$(cat resources.json | jq .cloudwatch_log_policy | tr -d '"')
+IAM_ROLE=$(cat resources.json | jq .iam_role | tr -d '"')
+EVENTBRIDGE_RULE=$(cat resources.json | jq .eventbridge_rule | tr -d '"')
+wait
+
+echo "Setting up deployment variables..."
+if [ $SUFFIX == null ] || [ $SUFFIX == "" ]; then
+    SUFFIX=$(date +%d%m%y)
+    RESOURCES_JSON=$(cat resources.json)
+    jq --arg suffix "${SUFFIX}" '.resource_suffix |= $suffix' <<< $RESOURCES_JSON > resources.json
+fi
+FUNCTION_NAME=data-ingestion-${SUFFIX}
+AWS_ACCOUNT_ID=$(aws sts get-caller-identity | jq .Account | tr -d '"')
+CODE_BUCKET_NAME=code-bucket-${SUFFIX}
+INGESTION_BUCKET=ingested-data-bucket-1
+wait
+
+echo "Generating AWS secret for source database credentials..."
+aws secretsmanager create-secret --name source-db-creds --secret-string file://__source_creds.json
+aws secretsmanager create-secret --name destination-db-creds --secret-string file://__destination_creds.json
+wait
+
+echo "Creating bucket code bucket '${CODE_BUCKET_NAME}'..."
+aws s3 mb s3://${CODE_BUCKET_NAME}
+wait
+
+echo "Creating function deployment package..."
+cd ../src/ingestion-folder/
+zip ../../deployment/__test-ingestion.zip __test-ingestion.py 
```

Comment:
Received error that '__test-ingestion.py' does not match any existing files, looks to be due to mismatched file names

Reply:
This was due to having the __test-ingestion.py in the .gitignore, I have renamed these and ensured the correct files are uploaded to the repo.